src/SFA.DAS.Roatp.CourseManagement.Web/Properties/launchSettings.json

```diff
@@ -20,7 +20,7 @@
       "launchBrowser": true,
       "applicationUrl": "https://localhost:5011;http://localhost:5010",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development"
+        "ASPNETCORE_ENVIRONMENT": "Production"
       }
```

Comment:
I think this needs to change back to 'Development'

Reply:
Done